.github/workflows/ipa-image-build.yml

```
@@ -25,11 +32,19 @@ env:
   ANSIBLE_FORCE_COLOR: True
   KAYOBE_ENVIRONMENT: ci-builder
   KAYOBE_VAULT_PASSWORD: ${{ secrets.KAYOBE_VAULT_PASSWORD }}
+    
 jobs:
+  runner-selection:
+    uses: ./.github/workflows/runner-selector.yml
+    with:
+      runner_env: ${{ inputs.runner_env }}
   ipa-image-build:
```

nit: whitespace could be improved
```suggestion
jobs:
  runner-selection:
    uses: ./.github/workflows/runner-selector.yml
    with:
      runner_env: ${{ inputs.runner_env }}

  ipa-image-build:
```